cmd/codegen/app/generate_clientset.go

```diff
@@ -83,7 +83,7 @@ func NewGenerateClientSetCmd(genOpts GenerateOptions) *cobra.Command {
 	cobraCmd.Flags().StringVarP(&o.InputPackage, optionInputPackage, "i", "", "Input package, must specify")
 	cobraCmd.Flags().StringVarP(&o.OutputPackage, optionOutputPackage, "o", "", "Output package, must specify")
 	cobraCmd.Flags().StringVarP(&o.InputBase, optionInputBase, "", wd, "Input base, defaults working directory")
-
+	cobraCmd.Flags().BoolVarP(&o.NotIsolated, optionNotIsolated, "", false, "do not isolate from the users GOPATH")
```

Comment:
Why is the option negative? I find "isolated" easier to process/understand.

Reply:
Should be on by default, so did the negative. TBH I'm not even sure this should be optional, I put this in mostly for you ;-)

Reply:
will change it to global